### PR TITLE
[feature](show) display storage medium of be by show proc

### LINKED ISSUE #14337: [Feature] show disk info by show backends
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Description

be 通过 be.conf的storage_root_path参数配置了不同的存储目录，但是目前没有很好的直观的方法看到各be的配置

建议可以在show backends的时候，增加一列，diskinfo，然后把配置的存储目录信息显示出来

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by nextdreamblue**
Signed-off-by: nextdreamblue <zxw520blue1@163.com>

# Proposed changes

Issue Number: close #14337

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [x] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.09 seconds
 load time: 440 seconds
 storage size: 17171603440 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221117081742_clickbench_pr_47900.html

File: fe/fe-core/src/test/java/org/apache/doris/utframe/DemoMultiBackendsTest.java [hunk 1]
```diff
@@ -201,9 +201,10 @@ public void testCreateDbAndTable() throws Exception {
         Assert.assertEquals(BackendsProcDir.TITLE_NAMES.size(), result.getColumnNames().size());
         Assert.assertEquals("{\"location\" : \"default\"}", result.getRows().get(0).get(20));
         Assert.assertEquals("{\"lastSuccessReportTabletsTime\":\"N/A\",\"lastStreamLoadTime\":-1,\"isQueryDisabled\":false,\"isLoadDisabled\":false}",
-                result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 3));
-        Assert.assertEquals("0", result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 2));
-        Assert.assertEquals(Tag.VALUE_MIX, result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 1));
+                result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 4));
+        Assert.assertEquals("0", result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 3));
+        Assert.assertEquals(Tag.VALUE_MIX, result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 2));
+        Assert.assertEquals("/home/doris.SSD:SSD; /home/doris.HDD:HDD; ", result.getRows().get(0).get(BackendsProcDir.TITLE_NAMES.size() - 1));
```
Comment (stalary) [inline fe/fe-core/src/test/java/org/apache/doris/utframe/DemoMultiBackendsTest.java:207]: `/home/doris` Whether depends on DORIS_HOME？

Comment (nextdreamblue) [reply fe/fe-core/src/test/java/org/apache/doris/utframe/DemoMultiBackendsTest.java:207]: it depends on be's conf: storage_root_path

> `/home/doris` Whether depends on DORIS_HOME？

**Review by stalary — DISMISSED** [review]
LGTM

**Review by morningman — CHANGES_REQUESTED** [review]
We can view the path by `show proc "/backends/10001"`, where 10001 is the id of the backend.
So I think we don't need to add this column to `show backends`.
And there maybe many disks for a certain backends, which cause the result too long.

**Comment by nextdreamblue** [comment]
> We can view the path by `show proc "/backends/10001"`, where 10001 is the id of the backend. So I think we don't need to add this column to `show backends`. And there maybe many disks for a certain backends, which cause the result too long.

it is good enough  by show proc "/backends/10001". this pr will colse

**Comment by nextdreamblue** [comment]
> We can view the path by `show proc "/backends/10001"`, where 10001 is the id of the backend. So I think we don't need to add this column to `show backends`. And there maybe many disks for a certain backends, which cause the result too long.

maybe can add a column to display medium info by `show proc "/backends/10001"`?

**Pull request closed without merging**